src/ess/isissans/mantidio.py

```diff
@@ -99,12 +100,28 @@ def from_data_workspace(
 
 
 def load_run(filename: Filename[RunType], period: Period) -> DataWorkspace[RunType]:
-    loaded = _mantid_simpleapi.Load(
-        Filename=str(filename), LoadMonitors=True, StoreInADS=False
-    )
+    # Loading many small files with Mantid is, for some reason, very slow when using
+    # the default number of threads in the Dask threaded scheduler (1 thread worked
```

Comment:
Can it be because multi-threading is also used internally by Mantid when it is loading the data?

Reply:
I think I checked when I first wrote this (note this code and comment was simply moved in this PR). I don't think so.